README.md

```diff
@@ -30,20 +54,16 @@ To push the built image to quay.io run:
 
 ```bash
 make image/push
-``` 
-
-## Ansible Tower Installation
+```
 
-The `install_tower.yml` playbook will install Ansible Tower on a target Openshift cluster.
+## 3. Ansible Tower Installation
 
-The playbook requires a number of configs to be passed in, namely:
+The `install_tower.yml` playbook will install Ansible Tower on a target Openshift cluster. The playbook requires the target tower environment to be specified.
 
-* `tower_openshift_master_url`: The hostname of the target Openshift cluster
-* `tower_openshift_username`: The username to be used to login to the Openshift cluster (requires admin permissions)
-* `tower_openshift_password`: The password of the Openshift user specified
+* `tower_environment`: The Ansible Tower environment (dev/test/qe etc.)
 
 ```bash
-ansible-playbook -i inventories/hosts playbooks/install_tower.yml -e tower_openshift_master_url=<tower_openshift_master_url> -e tower_openshift_username=<tower_openshift_username> -e tower_openshift_password=<tower_openshift_password>
+ansible-playbook -i <path-to-local-credentials-project>/inventories/hosts playbooks/install_tower.yml -e tower_environment=<tower-environment> --ask-vault-pass
```

Comment:
I actually don't think tower_environment is necessary here. I think the values will all just come from [here](https://github.com/integr8ly/tower_dummy_credentials/blob/master/inventories/group_vars/all/tower_openshift_authentication.yml). Could be worth recommending to put the tower_openshift_master_url in as an extra var each time to ensure the user is specifying the right openshift cluster?